include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -14,8 +14,77 @@
 
 namespace inexor::vulkan_renderer::wrapper {
 
-/// @brief A RAII wrapper class for VkDevice, VkPhysicalDevice and VkQueues.
-/// @note There is no method ``is_layer_supported`` in this wrapper class because device layers are deprecated.
+/// Call vkEnumerateDeviceExtensionProperties
```

Comment:
Not entirely sure if the wrapper namespace is the right one.

Reply:
Where would you put it?

Reply:
Probably in an own namespace or as static function in Device. Anyway those are not really functions in relation to wrapper.

Reply:
Done. I moved it to a separate file `vk_tools/enumerate.hpp`